scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -266,19 +266,22 @@ object TreeSyntax {
         }
         looksLikePatVar && thisLocationAlsoAcceptsPatVars
       }
-      // soft keywords might need to be written with backquotes in some places
+      /* Soft keywords might need to be written with backquotes in some places.
+       * Previously used match clause fails due to:
+       * https://github.com/scala-native/scala-native/issues/2187
```

Comment:
Interesting! Nice job with finding a minimal repro!

Reply:
that was all @WojciechMazur :tada: